pkg/triton/triton.go

```diff
@@ -177,10 +177,15 @@ func (ts *triton) ModelInferRequest(ctx context.Context, task commonPB.Task, inf
 				Shape:    []int64{1},
 			})
 		case commonPB.Task_TASK_TEXT_GENERATION:
+			inputShape := []int64{1}
+			if len(modelMetadata.Inputs[i].Shape) != 1 {
+				inputShape = append(inputShape, 1) // WITH BATCH SIZE 1
```

Comment:
though it will always be (1, 1) in this case, but `inputShape` dimension should be swapped to avoid confusion.
first dimension is batch size

Reply:
Based on the [discussion here](https://github.com/vllm-project/vllm/issues/541), it appears that vLLM has a setup where batch processing is managed internally, rather than through the Triton server. I've updated the code to determine if the inference requires Triton server's batch mechanism based on the config.